### ISSUE TITLE
"ssh: command not found" when connecting to host

### ISSUE THREAD

**Issue by legenddcr** [microsoft/vscode-remote-release#2294]
Version: 1.41.1
Commit: 26076a4de974ead31f97692a0d32f90d735645c0
Date: 2019-12-18T14:57:51.166Z
Electron: 6.1.5
Chrome: 76.0.3809.146
V8: 7.6.303.31-electron.0
OS: Darwin x64 19.2.0

Steps to Reproduce:

Remote-SSH: connect to Host
Select Host
Get "Could not establish connection to [Host]" error with following trace in output:

```
[12:17:36.970] remote-ssh@0.48.0
[12:17:36.970] darwin x64
...
[12:17:36.997] Testing ssh with ssh -V
[12:17:37.015] ssh exited with code: 0
[12:17:37.015] Got stderr from ssh: OpenSSH_7.9p1, LibreSSL 2.7.3
[12:17:37.019] Running script with connection command: ssh -T -D 59540 -o ConnectTimeout=15  bash
[12:17:37.019] Install and start server if needed
[12:17:37.171] > /bin/sh: ssh: command not found
[12:17:37.171] Got some output, clearing connection timeout
[12:17:37.606] "install" terminal command done
[12:17:37.606] Install terminal quit with output: /bin/sh: ssh: command not found
[12:17:37.606] Received install output: /bin/sh: ssh: command not found
[12:17:37.607] Stopped parsing output early. Remaining text: /bin/sh: ssh: command not found
[12:17:37.607] Failed to parse remote port from server output
[12:17:37.607] Resolver error: 
[12:17:37.611] ------
```

Several similar issues are reported in the past, e.g. https://github.com/microsoft/vscode-remote-release/issues/1853. It is found my ssh on /user/bin/ssh and can work in host environment correctly.

**Comment by roblourens** [comment]
Is `ssh` on your path? If you run `ssh` in an external terminal, is it found?

**Comment by 0x1abin** [comment]
I have the same problem on Mac OS.
run ssh in an external terminal is found.

**Comment by roblourens** [comment]
It looks like ssh is not on your PATH, could it be that the external terminal has the PATH modified to include it?

**Comment by 0x1abin** [comment]
Resolved in the latest version 1.42.0.

**Comment by roblourens** [comment]
Great - closing unless I get an update from the OP
